testing_guide.md

```diff
@@ -0,0 +1,698 @@
biI6WzU1Ni45NTE5NTcxNzE2NDE0LDEyNC42MTE0ODQ0NTUzODg2Ml0sIm5hbWUiOiJEZXNjcmlwdG9yIn0sIjgiOnsiaWQiOjgsImRhdGEiOnsibnVtIjoyMX0sImlucHV0cyI6e30sIm91dHB1dHMiOnsicG9sIjp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNywiaW5wdXQiOiJwb2wxIiwiZGF0YSI6e319XX19LCJwb3NpdGlvbiI6Wy03OC41NTk4NTE4NjQzMzg0OCwtNjUuNTQ3MDM1NjYxMDU4NDZdLCJuYW1lIjoiQWZ0ZXIifSwiMTAiOnsiaWQiOjEwLCJkYXRhIjp7fSwiaW5wdXRzIjp7ImtleSI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6MTIsIm91dHB1dCI6ImtleSIsImRhdGEiOnt9fV19fSwib3V0cHV0cyI6eyJrZXkiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE2LCJpbnB1dCI6InBvbGljaWVzIiwiZGF0YSI6e319XX19LCJwb3NpdGlvbiI6Wy0zNDEuMjAzMDYwNTIzMDI2Myw0OTYuNTM3NjQ5NTU1NjA3NF0sIm5hbWUiOiJLZXkifSwiMTIiOnsiaWQiOjEyLCJkYXRhIjp7Im1uZW1vbmljIjoic3BlbGwgYm9yZGVyIGV4aGF1c3QgeWFyZCBlY29ub215IGZvc3NpbCBhcm15IGd1ZXNzIHR5cGljYWwgYm9tYiBhY2hpZXZlIGFtb3VudCJ9LCJpbnB1dHMiOnt9LCJvdXRwdXRzIjp7ImtleSI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6MTAsImlucHV0Ijoia2V5IiwiZGF0YSI6e319XX19LCJwb3NpdGlvbiI6Wy02NzYuNzgwMTk0NTc5Mzg1NCw0MzMuMTY2OTMxMDkzMDg4MjRdLCJuYW1lIjoiQklQMzkifSwiMTQiOnsiaWQiOjE0LCJkYXRhIjp7Im1uZW1vbmljIjoiY2FtZXJhIHVuYWJsZSB0cmljayBpbmNvbWUgd2lzZG9tIG1peCByZW1pbmQgb3N0cmljaCBvZmZpY2UgbWFzdGVyIHN3YW1wIGxvdHRlcnkifSwiaW5wdXRzIjp7fSwib3V0cHV0cyI6eyJrZXkiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE1LCJpbnB1dCI6ImtleSIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstMTAyNi4wNzIwNjE4NDk4NzY3LDI2NS41NzE0OTg4ODQxOTIzXSwibmFtZSI6IkJJUDM5In0sIjE1Ijp7ImlkIjoxNSwiZGF0YSI6e30sImlucHV0cyI6eyJrZXkiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE0LCJvdXRwdXQiOiJrZXkiLCJkYXRhIjp7fX1dfX0sIm91dHB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNiwiaW5wdXQiOiJwb2xpY2llcyIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstMzYyLjQzMzExNjAxNTIxNDIsMjM3LjE3NDY1OTM5MzAyOTRdLCJuYW1lIjoiS2V5In0sIjE2Ijp7ImlkIjoxNiwiZGF0YSI6eyJ0aHJlc2giOjJ9LCJpbnB1dHMiOnsicG9saWNpZXMiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjEwLCJvdXRwdXQiOiJrZXkiLCJkYXRhIjp7fX0seyJub2RlIjoxNSwib3V0cHV0Ijoia2V5IiwiZGF0YSI6e319XX19LCJvdXRwdXRzIjp7InBvbCI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6MTcsImlucHV0IjoicG9sMiIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstNDIuNzg0Nzc3NTQ4NTc2NzQsMjUyLjIyMjU2NzM5MjgyMjAxXSwibmFtZSI6IlRocmVzaG9sZCJ9LCIxNyI6eyJpZCI6MTcsImRhdGEiOnt9LCJpbnB1dHMiOnsicG9sMSI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6OCwib3V0cHV0IjoicG9sIiwiZGF0YSI6e319XX0sInBvbDIiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE2LCJvdXRwdXQiOiJwb2wiLCJkYXRhIjp7fX1dfX0sIm91dHB1dHMiOnsicG9sIjp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjo2LCJpbnB1dCI6InBvbCIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlsyNTEuNjQ5NzA2MzcyNTA5MiwxMTEuMzMyODE0NzQ5MjcxNTVdLCJuYW1lIjoiQW5kIn19LCJjb21tZW50cyI6W3sidGV4dCI6IkFsaWNlIiwicG9zaXRpb24iOlstMTA4OS4zODA5NDUzOTI1OTM3LDIwMC4yMjM3Njc5NzM4MjYxN10sImxpbmtzIjpbMTRdLCJ0eXBlIjoiZnJhbWUiLCJ3aWR0aCI6MjgwLCJoZWlnaHQiOjQxNS45OTk5OTk5OTk5OTk5NH0seyJ0ZXh0IjoiQm9iIiwicG9zaXRpb24iOlstNzQzLjgyNTA0MTQzMzI4MjQsLTE1OS41NDIwOTc0MjI1MzU2OF0sImxpbmtzIjpbMl0sInR5cGUiOiJmcmFtZSIsIndpZHRoIjoyODAsImhlaWdodCI6NDE2fSx7InRleHQiOiJDYXJvbCIsInBvc2l0aW9uIjpbLTczOS4yMDk4NjkwODk1MTg0LDM2OS4yMDIxMjg0MjE0MTQxXSwibGlua3MiOlsxMl0sInR5cGUiOiJmcmFtZSIsIndpZHRoIjoyODAsImhlaWdodCI6NDE2fV0sIm5ldHdvcmsiOiJzaWduZXQifQ==
```

Comment:
Should this link also include a line that connects Bob's key to the threshold (multisig) policy? Here's what it looks like with that line added: https://miniscript.fun/#/full/eyJpZCI6ImRlbW9AMC4xLjAiLCJub2RlcyI6eyIyIjp7ImlkIjoyLCJkYXRhIjp7Im1uZW1vbmljIjoiY3Jhc2ggZmF0YWwgaG9sbG93IHRoYW5rIHN3YWxsb3cgc3VibWl0IHRhdHRvbyBwb3J0aW9uIGNvZGUgZm9hbSBtYXRoIGZvcmNlIiwicGFzc3dvcmQiOiIiLCJkZXJpdmF0aW9uIjoiIn0sImlucHV0cyI6e30sIm91dHB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjozLCJpbnB1dCI6ImtleSIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstNjgyLjk3ODA3NzAxNzExNTUsLTk0LjM1NDg3NDg2ODg2NzM4XSwibmFtZSI6IkJJUDM5In0sIjMiOnsiaWQiOjMsImRhdGEiOnt9LCJpbnB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoyLCJvdXRwdXQiOiJrZXkiLCJkYXRhIjp7fX1dfX0sIm91dHB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNiwiaW5wdXQiOiJwb2xpY2llcyIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstMzU0LjM4NTI2MTYwMTMyMjYsLTI2LjE3MDA1ODk3OTU4NDMyMl0sIm5hbWUiOiJLZXkifSwiNiI6eyJpZCI6NiwiZGF0YSI6e30sImlucHV0cyI6eyJwb2wiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE3LCJvdXRwdXQiOiJwb2wiLCJkYXRhIjp7fX1dfX0sIm91dHB1dHMiOnsiZGVzYyI6eyJjb25uZWN0aW9ucyI6W119fSwicG9zaXRpb24iOls1NTYuOTUxOTU3MTcxNjQxNCwxMjQuNjExNDg0NDU1Mzg4NjJdLCJuYW1lIjoiRGVzY3JpcHRvciJ9LCI4Ijp7ImlkIjo4LCJkYXRhIjp7Im51bSI6MjF9LCJpbnB1dHMiOnt9LCJvdXRwdXRzIjp7InBvbCI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6MTcsImlucHV0IjoicG9sMSIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstNzguNTU5ODUxODY0MzM4NDgsLTY1LjU0NzAzNTY2MTA1ODQ2XSwibmFtZSI6IkFmdGVyIn0sIjEwIjp7ImlkIjoxMCwiZGF0YSI6e30sImlucHV0cyI6eyJrZXkiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjEyLCJvdXRwdXQiOiJrZXkiLCJkYXRhIjp7fX1dfX0sIm91dHB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNiwiaW5wdXQiOiJwb2xpY2llcyIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstMzQxLjIwMzA2MDUyMzAyNjMsNDk2LjUzNzY0OTU1NTYwNzRdLCJuYW1lIjoiS2V5In0sIjEyIjp7ImlkIjoxMiwiZGF0YSI6eyJtbmVtb25pYyI6InNwZWxsIGJvcmRlciBleGhhdXN0IHlhcmQgZWNvbm9teSBmb3NzaWwgYXJteSBndWVzcyB0eXBpY2FsIGJvbWIgYWNoaWV2ZSBhbW91bnQifSwiaW5wdXRzIjp7fSwib3V0cHV0cyI6eyJrZXkiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjEwLCJpbnB1dCI6ImtleSIsImRhdGEiOnt9fV19fSwicG9zaXRpb24iOlstNjc2Ljc4MDE5NDU3OTM4NTQsNDMzLjE2NjkzMTA5MzA4ODI0XSwibmFtZSI6IkJJUDM5In0sIjE0Ijp7ImlkIjoxNCwiZGF0YSI6eyJtbmVtb25pYyI6ImNhbWVyYSB1bmFibGUgdHJpY2sgaW5jb21lIHdpc2RvbSBtaXggcmVtaW5kIG9zdHJpY2ggb2ZmaWNlIG1hc3RlciBzd2FtcCBsb3R0ZXJ5In0sImlucHV0cyI6e30sIm91dHB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNSwiaW5wdXQiOiJrZXkiLCJkYXRhIjp7fX1dfX0sInBvc2l0aW9uIjpbLTEwMjYuMDcyMDYxODQ5ODc2NywyNjUuNTcxNDk4ODg0MTkyM10sIm5hbWUiOiJCSVAzOSJ9LCIxNSI6eyJpZCI6MTUsImRhdGEiOnt9LCJpbnB1dHMiOnsia2V5Ijp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNCwib3V0cHV0Ijoia2V5IiwiZGF0YSI6e319XX19LCJvdXRwdXRzIjp7ImtleSI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6MTYsImlucHV0IjoicG9saWNpZXMiLCJkYXRhIjp7fX1dfX0sInBvc2l0aW9uIjpbLTM2Mi40MzMxMTYwMTUyMTQyLDIzNy4xNzQ2NTkzOTMwMjk0XSwibmFtZSI6IktleSJ9LCIxNiI6eyJpZCI6MTYsImRhdGEiOnsidGhyZXNoIjoyfSwiaW5wdXRzIjp7InBvbGljaWVzIjp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxMCwib3V0cHV0Ijoia2V5IiwiZGF0YSI6e319LHsibm9kZSI6MTUsIm91dHB1dCI6ImtleSIsImRhdGEiOnt9fSx7Im5vZGUiOjMsIm91dHB1dCI6ImtleSIsImRhdGEiOnt9fV19fSwib3V0cHV0cyI6eyJwb2wiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjE3LCJpbnB1dCI6InBvbDIiLCJkYXRhIjp7fX1dfX0sInBvc2l0aW9uIjpbLTQyLjc4NDc3NzU0ODU3Njc0LDI1Mi4yMjI1NjczOTI4MjIwMV0sIm5hbWUiOiJUaHJlc2hvbGQifSwiMTciOnsiaWQiOjE3LCJkYXRhIjp7fSwiaW5wdXRzIjp7InBvbDEiOnsiY29ubmVjdGlvbnMiOlt7Im5vZGUiOjgsIm91dHB1dCI6InBvbCIsImRhdGEiOnt9fV19LCJwb2wyIjp7ImNvbm5lY3Rpb25zIjpbeyJub2RlIjoxNiwib3V0cHV0IjoicG9sIiwiZGF0YSI6e319XX19LCJvdXRwdXRzIjp7InBvbCI6eyJjb25uZWN0aW9ucyI6W3sibm9kZSI6NiwiaW5wdXQiOiJwb2wiLCJkYXRhIjp7fX1dfX0sInBvc2l0aW9uIjpbMjUxLjY0OTcwNjM3MjUwOTIsMTExLjMzMjgxNDc0OTI3MTU1XSwibmFtZSI6IkFuZCJ9fSwiY29tbWVudHMiOlt7InRleHQiOiJBbGljZSIsInBvc2l0aW9uIjpbLTEwODkuMzgwOTQ1MzkyNTkzNywyMDAuMjIzNzY3OTczODI2MTddLCJsaW5rcyI6WzE0XSwidHlwZSI6ImZyYW1lIiwid2lkdGgiOjI4MCwiaGVpZ2h0Ijo0MTUuOTk5OTk5OTk5OTk5OTR9LHsidGV4dCI6IkJvYiIsInBvc2l0aW9uIjpbLTc0My44MjUwNDE0MzMyODI0LC0xNTkuNTQyMDk3NDIyNTM1NjhdLCJsaW5rcyI6WzJdLCJ0eXBlIjoiZnJhbWUiLCJ3aWR0aCI6MjgwLCJoZWlnaHQiOjQxNn0seyJ0ZXh0IjoiQ2Fyb2wiLCJwb3NpdGlvbiI6Wy03MzkuMjA5ODY5MDg5NTE4NCwzNjkuMjAyMTI4NDIxNDE0MV0sImxpbmtzIjpbMTJdLCJ0eXBlIjoiZnJhbWUiLCJ3aWR0aCI6MjgwLCJoZWlnaHQiOjQxNn1dLCJuZXR3b3JrIjoic2lnbmV0In0=

Reply:
Yes, you are right! Which also makes the resulting output descriptor in the guide wrong.